pkg/stdlib/pkgs/query.cue

```diff
@@ -124,3 +124,29 @@
 	}]
 	...
 }
+
+#GetApplicationTree: {
+	#do:       "getApplicationTree"
+	#provider: "query"
+	app: {
+		name:      string
+		namespace: string
```

Comment:
miss the filter option?

Reply:
I don't get the scenario. Why we need the filter when show a application tree?